README.md

```diff
@@ -13,8 +13,8 @@ This project contains core functionality required by Node.js code generated by t
 ```
 
 ## Prerequisites
-- Node.js version 14 or newer
-- Note that v14 is technically *supported* but not officially *guaranteed* as we only test on v16 and higher
+- Node.js version 18 or newer
+- Note that v18 is technically *supported* but not officially *guaranteed* as we only test on v16 and higher
```

Comment:
We can remove this line - our minimum version and our build have been aligned.

Reply:
done